deploy/helm/flightctl/templates/flightctl-cleanup-job.yaml

```diff
@@ -25,15 +25,58 @@ spec:
           args:
             - |
               set -e
-              echo "Starting post-delete cleanup process..."
-              echo "Deleting orphaned PVCs..."
+              TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+              CACERT=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+              API="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}"
+
+              # --- JQ Installation ---
+              # Define jq version and download URL (check for latest stable version on GitHub)
+              # As of today, July 30, 2025, 1.7.1 is the latest stable.
+              JQ_VERSION="1.7.1"
+              JQ_URL="https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux-amd64"
```

Comment:
Wouldn't this approach fail in a disconnected environment?

Reply:
I guess the only other option is using a combination of field selector, grep, and sed.